tendermint/src/evidence.rs

```diff
@@ -92,5 +92,37 @@ pub struct Params {
         serialize_with = "serializers::serialize_u64",
         deserialize_with = "serializers::parse_u64"
     )]
-    pub max_age: u64,
+    pub max_age_num_blocks: u64,
+
+    /// Max age duration
+    pub max_age_duration: Duration,
+}
+
+/// Duration
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct Duration(u64);
```

Comment:
Can we add some comments for why we need this (eg. compared to max_age_num_blocks, which is also just a u64)

Reply:
@ebuchman I've addressed this [here](https://github.com/informalsystems/tendermint-rs/pull/196/commits/f0048c58f3764a3a29beec4cba0c81b43cd14f84#diff-3b2ccc4b9e2bf7cc4225b022246b6642R101)